src/Compilers/VisualBasic/Portable/Binding/LookupResult.vb

```diff
@@ -722,6 +728,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             MergeAmbiguous(other, s_ambiguousInTypeError)
         End Sub
 
+        Private Function AreEquivalentEnumConstants(symbol1 As Symbol, symbol2 As Symbol) As Boolean
+            If symbol1.Kind <> SymbolKind.Field OrElse symbol2.Kind <> SymbolKind.Field OrElse symbol1.ContainingType.TypeKind <> TypeKind.Enum Then
+                Return False
+            End If
+            Dim f1 = CType(symbol1, FieldSymbol)
+            Dim f2 = CType(symbol2, FieldSymbol)
+            Return f1.ConstantValue IsNot Nothing AndAlso f1.ConstantValue.Equals(f2.ConstantValue)
```

Comment:
As we discussed offline, "imported" here doesn't mean "not from source", but it at least means "from different compilation". So, we  indeed should be safe in terms of circularities.